README.md

```diff
@@ -1,6 +1,12 @@
 Random User Agents
 ==================
 
+Use this command to install this forked version:
```

Comment:
Please revert these documentation updates. After the PR merge these changes will become part of the documentation and will misguide the developers.